config/openloader/data/HLTweaker/data/hltweaker/tags/blocks/needs_ancient_elements_tier_eight_tool.json

```diff
@@ -0,0 +1,6 @@
+{
+  "values": [
```

Comment:
Tags for blocks to display correct mining level in Jade https://github.com/yzl210/HarvestLevelTweaker?tab=readme-ov-file#set-block-harvest-level